curriculum/challenges/english/08-coding-interview-prep/algorithms/implement-insertion-sort.english.md

```diff
@@ -41,14 +41,10 @@ tests:
 
 ```js
 function insertionSort(array) {
-  // change code below this line
```

Comment:
Please do not remove `// change code below this line` from the challenge seed code.